### ISSUE TITLE
'High Contrast Black' theme List WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1009]
**Describe the bug**
The List components' is-selected state color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/list.html
2. Check List's is-selected state color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81749811-2d3e0800-9461-11ea-9dbe-b3d451b271f3.png)
![image](https://user-images.githubusercontent.com/43764832/81756123-b52b0e80-946f-11ea-99a1-44e7a8cac1f2.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
